skglm/utils.py

```diff
@@ -89,11 +89,11 @@ def compiled_clone(instance, to_float32=False):
 
 
 @njit
-def ST(x, u):
+def ST(x, u, positive):
```

Comment:
We better make `positive` an optional argument (`positive=False`)?

- It won't impact other parts of the code that uses `ST`
- It's not intuitive to pass in a 3rd argument to ST